src/CameraKitCamera.android.js

```diff
@@ -8,6 +8,8 @@ import {
 
 const NativeCamera = requireNativeComponent('CameraView', null);
 const NativeCameraModule = NativeModules.CameraModule;
+const TORCH_MODE_ON = "on";
+const TORCH_MODE_CALL_ARG = "torch";
```

Comment:
plz use a single quote for strings `'` instead of `"`

Reply:
done